pipe/runtime/src/sections/snowflake/destination.rs

```diff
@@ -124,7 +122,9 @@ impl SnowflakeDestination {
         writer.close().await?;
 
         // todo: use load and select into custom stage
```

Comment:
the table name substitution we're doing a few lines down below is a hacky solution. I believe this comment about a custom stage would help us solve the table name issue. if I understand the docs correctly, if we use a custom stage we would have more flexibility and could use the actual origin as the table name as long as it is enclosed in quotes.